libmscore/tempotext.cpp

```diff
@@ -236,7 +236,7 @@ QVariant TempoText::propertyDefault(P_ID id) const
       {
       switch(id) {
             case P_ID::TEMPO:             return 2.0;
-            case P_ID::TEMPO_FOLLOW_TEXT: return false;
+            case P_ID::TEMPO_FOLLOW_TEXT: return true;
```

Comment:
because if you take a tempo text from the palette, it has the 'reset value' button enabled. I think TEMPO_FOLLOW_TEXT being set to true is a sane default. Setting it in constructor too though,causes many test failures and you (rightfully) complained about those ;-)